app.json

```diff
@@ -21,10 +21,16 @@
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "package": "com.alilari1986.CirclesApp"
```

Comment:
remove `alilari1986` from the package name

Reply:
Done.